CHANGELOG.md

```diff
@@ -51,6 +51,9 @@
 
 * `[jest-config]` Add `forceCoverageMatch` to allow collecting coverage from
   ignored files. ([#5081](https://github.com/facebook/jest/pull/5081))
+* `[jest-cli]` Make Jest exit without an error when no tests are found in
```

Comment:
can you move this to be under `master` at the top?

You probably need a rebase